src/EditorFeatures/CSharpTest/SignatureHelp/GenericNameSignatureHelpProviderTests.cs

```diff
@@ -849,5 +850,44 @@ void Goo()
             var expectedOrderedItems = new List<SignatureHelpTestItem>();
             await TestAsync(markup, expectedOrderedItems);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        public async Task DeclaringGenericTypeWithDocCommentList()
```

Comment:
Consider adding a WorkItem for the linked issue.

Reply:
Good point, thanks!